src/Reducer/tokenClient.js

```diff
@@ -0,0 +1,12 @@
+const initialState = {
+  tokenClient: "0",
```

Comment:
I'm not sure that's the best initial value for a token: could easily defeat an `if` in your code
Other, better values: `null`, `undefined`, `""`